README.md

```diff
@@ -23,31 +27,130 @@ const job = await client.createReport({
   endTime: new Date().toISOString(),
   browserName: "Chrome",
   browserVersion: "105",
-  framework: "playwright",
+  framework: "playwright", // alternatives: cypress, playwright, testcafe
   frameworkVersion: "1.25.0",
   platformName: "Windows 11"
 });
 console.log(job.id); // the job ID
 console.log(job.url); // the full URL of the job
 ```
 
-## Upload Assets
+### Upload Log
+
+In order to view your test results in the SauceLabs web app, you must upload a test report file along with the report created in the step above. You can use the [sauce-json-reporter](https://github.com/saucelabs/sauce-json-reporter-js) library to generate the test report programatically or manually create the json file according to the report schema defined [here](https://github.com/saucelabs/sauce-json-reporter-js/blob/main/api/schema.json).
```

Comment:
```suggestion
In order to view your test results in the SauceLabs web app, you must upload a test report file along with the report created in the step above. You can use the [sauce-json-reporter](https://github.com/saucelabs/sauce-json-reporter-js) library to generate the test report programmatically or manually create the json file according to the report schema defined [here](https://github.com/saucelabs/sauce-json-reporter-js/blob/main/api/schema.json).
```

Reply:
Huh, I've been misspelling that my whole career, apparently. 🤦